### ISSUE TITLE
EDGE-BASED LOSS FUNCTION FOR SINGLE IMAGE SUPER-RESOLUTION

### ISSUE THREAD

**Issue by DIPTIMISHRA** [GeorgeSeif/VDSR-Keras#5]
Hi,
can you add code for the mentioned paper,
basically wanted to have this edge based loss function

Thanks in advance
You can also share the code on

dipti.mishra28@gmail.com
